packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -87,6 +98,11 @@ export class ObjectForest implements IEditableForest {
 	}
 
 	public constructor(
+		public readonly breaker: Breakable,
+		/**
+		 * If provided, may be used for validation, but otherwise unused.
+		 */
+		public readonly schema: TreeStoredSchemaSubscription | undefined,
```

Comment:
When is this expected to be undefined? Seems odd to have `additionalAsserts` enabled, but have no validation happen anyways.

Reply:
Some tests, especially when not using additionalAsserts, might not provide it.

Its nice for separation of concerns reasons to allow some minimal forest testing without involving all the schema stuff.

Also note that even if you don't provide schema, forks of this forest will get some and validate it, and some day we should have more asserts than just these which will still be worth opting into.

Also its no odder than before this change where additionalAsserts always did nothing (again in anticipation of future validation being added)

Reply:
> Some tests, especially when not using additionalAsserts, might not provide it.

Is it just test code that doesn't provide schema here? Just wondering if we should ultimately make this a required property. Otherwise, I worry we will introduce production uses that omit this by mistake and lose out on coverage.

Reply:
Long term (after chunked forest is default, which is a requirement for incremental summary) this entire forest is intended only as a reference implementation for testing and debugging.

The production use for this is for customers doing testing/validation.

Regardless, it's a valid concern so I have refactored the `Initialize invalid content` tests which have converge to ensure this works. It now uses the same user facing APIs for configuring the forest, so the test serves as an integration test to guard against such a change.